### DISCUSSION TITLE
Does Micropython esp32-s3 support camera?

### DISCUSSION THREAD

**Discussion by fary99** [micropython/micropython#12723] [ESP32]

**Comment by sophiedegran** [comment]
Shareitumin have done the job for many cameras but he propose only the firmware... you can have a look on github.com/shareiltumin/
-T

**Comment by shariltumin** [comment]
MP firmwares for ESP32-S3 that support OV2640 can be found [here](https://github.com/shariltumin/esp32-cam-micropython-2022/tree/main/firmwares-20230717/ESP32-S3).
